lib/lightning_web/live/workflow_live/index.ex

```diff
@@ -220,4 +324,18 @@ defmodule LightningWeb.WorkflowLive.Index do
         error_msg
     end
   end
+
+  defp to_sort_key("name"), do: :name
+  defp to_sort_key("enabled"), do: :enabled
+  defp to_sort_key("workorders_count"), do: :workorders_count
+  defp to_sort_key("failed_workorders_count"), do: :failed_workorders_count
+  defp to_sort_key("last_workorder_updated_at"), do: :last_workorder_updated_at
+  defp to_sort_key(_), do: :name
+
+  defp to_sort_direction("asc"), do: :asc
+  defp to_sort_direction("desc"), do: :desc
+  defp to_sort_direction(_), do: :asc
```

Comment:
@elias-ba For `nil`, I am happy to go with `:asc` but better not use the generic matching because it might hide some bug.

Reply:
I explicitly added a case for nil